src/components/dialogs/reactive-capability-curve-table.js

```diff
@@ -21,9 +21,10 @@ export const ReactiveCapabilityCurveTable = ({
     inputForm,
     isFieldRequired,
     disabled = false,
+    customPLabel = undefined,
```

Comment:
Why do you need a customPLabel ?

Reply:
This is to override the default "P" label in the field. It is needed for the first and last lines of the reactive capability limit curves.